### PR TITLE
Convert to JSX

### PR DESCRIPTION
**Pull request by quocpham2110**

### CODE REVIEW

**Comment by rindk** [comment]
ready

**Comment by rindk** [comment]
ready

**Comment by quocpham2110** [comment]
ready

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="theme-color" content="#000000" />
+  <meta name="description" content="Web site created using create-react-app" />
+  <title>E-commerce</title>
```
Comment (chienpv-0644) [inline public/index.html:14]: Tìm hiểu và cài đặt I18n luôn nhé

**Comment by quocpham2110** [comment]
ready

**Comment by quocpham2110** [comment]
- convert from HTML to JXS
- only converting and config Router for header and menu
- functions will be add at next commit

**Comment by quocpham2110** [comment]
ready

**Pull request merged**